### ISSUE TITLE
Part 3.1 Script in video does not match script in repository

### ISSUE THREAD

**Issue by rvrlx** [hposton/python-for-cybersecurity#7]
The script inside the Infosec Github repo, named "test_default_creds.py" does not match the script which is demonstrated
in the accompanying video. The version in the video is also named differently, i.e. "TestDefaultCredentials.py".

TestDefaultCredentials.py appears to be an improved version. 

Both scripts do essentially the same thing. However, there are significant differences between the script in the repo and the script in the video, which may negatively impact a beginner's learning experience.

**Comment by ianp89** [comment]
Yes, specifically the telnet function is quite different, and I believe there is a one-line difference in the SSH tester.
